SDAPackageWithVectorField/SDAWithVectorField/obstacle_map.py

```diff
@@ -0,0 +1,64 @@
+import numpy
+from math import atan2, cos, sin, pi, sqrt
+from SDAWithVectorField import *
+
+class ObstacleMap(object):
+	def __init__(self, drone_point, fly_zones):
+		self.obstacles = numpy.array([])
+		self.drone = Drone(drone_point, numpy.array([]))
+		self.fly_zones = FlightBoundariesContainer(fly_zones)
+		self.repulsive_forces = numpy.array([0,0,0])
+		self.attractive_force = numpy.array([0,0,0])
```

Comment:
You do not need to declare the variables here; instead, you can just call the ```reset_attractive_force()``` and ```reset_repulsive_forces()``` methods here.